src/macropad/index.html

```diff
@@ -0,0 +1,175 @@
+	<li><a href="#links">Links</a></li>
+        <li><a href="#parts_list">Parts List</a></li>
+        <li><a href="#build_guide">Build Guide</a></li>
+      </ul>
+
+      <h2 id="macropad">Overview</h2>
+      <figure>
+        <img src="macropad.jpg" width="600" height="657" />
+        <figcaption>PDXKBC macropad designed and built by Franklin</figcaption>
+      </figure>
+
+      <h2 id="links">Links</h2>
+      <ul>
+        <li>
+          <a href="https://github.com/fharding1/pdxkbc-badge"
+            >Hardware GitHub repository</a
+          >
+        </li>
+        <li>
+          <a
+            href="https://github.com/qmk/qmk_firmware/tree/master/keyboards/pdxkbc"
+            >QMK GitHub repository</a
+          >
+        </li>
+      </ul>
+
+      <h2 id="parts_list">Parts List</h2>
+      <figure>
+        <img src="materials.jpg" width="600"/>
+      </figure>
+      <ul>
+        <li>
+          Badge and Case PCBs (See <a href="#links">Links</a> section for source files)
+        </li>
+        <li>
+          Standoffs:
+        </li>
+          <ul>
+            <li>4x M4 10+6mm brass Male-Female Spacers</li>
+            <li>4x M3 x 6mm screws</li>
+            <li>4x M3 nuts</li>
+          </ul>
+        <li>
+          1x 5v Arduino Pro Micro
+        </li>
+        <li>
+          6x 1N4148 Diodes
+        </li>
+        <li>
+          4x Rubber Bumpers/Feet
+        </li>
+        <li>
+          6x PCB Mount Switches
+        </li>
+        <li>
+          6x 1u Keycaps
+        </li>
+      </ul>
+
+      <h2 id="build_guide">Build Guide</h2>
+
+      <h3>Tools</h3>
+      <p>You'll need</p>
+      <ul>
+        <li>A soldering iron and solder</li>
+        <li>A phillips screwdriver</li>
+        <li>Flush cutters</li>
+      </ul>
+
+      <h3>Assembly</h3>
+      <h4>Diodes</h4>
+      <p>Bend your diodes as shown</p>
+      <figure>
+        <img src="bentdiode.jpg" width="600"/>
+      </figure>
+      <p>
+        Place your diodes with the black stripe on the diode facing the white
+        stripe on the PCB, slightly bend the legs to prevent the diode from
+        falling out.
+      </p>
+      <figure>
+        <img src="placeddiode.jpg" width="600"/>
+        <img src="diodes.jpg" width="600"/>
+      </figure>
+      <p>Solder the diodes, and trim the legs with flush cutters.</p>
+      <figure>
+        <img src="trimmeddiodes.jpg" width="600"/>
+      </figure>
+      <h4>Switches & Pro Micro</h4>
+      <p>Place and solder the lower four switches</p>
+      <figure>
+        <img src="fourswitches.jpg" width="600"/>
+      <figure>
+      <p>
+        Tack on the two pin headers, fiddle with them until they're straight
+        and the pro micro fits on.
+      </p>
+      <figure>
+        <img src="tackedheader.jpg" width="600"/>
+        <img src="testfit.jpg" width="600"/>
+      <figure>
+      <p>Solder the pin header.</p>
+      <figure>
+        <img src="solderedheaders.jpg" width="600"/>
```

Comment:
it's best to include the actual width and height of the source image in the HTML, as it allows the browser to preserve space correctly for all images to prevent the page from jumping around as they load